src/db/models/expenses.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../main');
+
+const Expenses = sequelize.define('Expense', {
+  userId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  title: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  category: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  amount: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  spentAt: {
```

Comment:
Consider using 'spent_at' as the key name instead of 'spentAt' to follow the snake_case naming convention for database columns 🐍